src/utilities/music/AudioListener.java

```diff
@@ -44,7 +44,7 @@ public boolean nextTrack(){
 	public void onTrackEnd(AudioPlayer player, AudioTrack track,
 			AudioTrackEndReason endReason){
 
-//		System.out.println(tracks.remainingCapacity());
+			this.player.getCommand().remember(track.makeClone(), "LATEST_SONG");
```

Comment:
That `track.makeClone()` should have the `.makeClone()` method added to it - we don't need a clone of the object in the buffer.